python/py-testtools/Portfile

```diff
@@ -29,7 +28,7 @@ checksums           rmd160  203aa69815a8b75b699f6f588431330aa2d3f072 \
 python.versions     27 37 38 39
 
 if {${name} ne ${subport}} {
-    if {${python.version} == 27} {
+    if {${python.version} eq {27}} {
```

Comment:
Steve, it looks like all of our Python ports always use the double-equals numeric comparison, rather than `eq`. Why the switch...?